src/conf_mode/service_ids_fastnetmon.py

```diff
@@ -55,8 +55,11 @@ def verify(fastnetmon):
     if 'mode' not in fastnetmon:
         raise ConfigError('Specify operating mode!')
 
-    if 'listen_interface' not in fastnetmon:
-        raise ConfigError('Specify interface(s) for traffic capture')
+    if fastnetmon.get('mode') == 'mirror' and 'listen_interface' not in fastnetmon:
+        raise ConfigError('Specify interface(s) for traffic capture mode mirror')
```

Comment:
This is a pretty confusing error message. What's a "traffic capture mode mirror"? A magical artifact ­— a mirror where you can see the traffic capture mode setting from your config at midnight under full moon? :)

I think it should be "Incorrect settings for 'mode mirror": must specify interface(s) for traffic mirroring".

Reply:
Done!